swagger2.0/oic.r.cloudopenapi.swagger.json

```diff
@@ -722,49 +722,50 @@
       }
     },
     "DeviceWithLinks": {
-      "allOf": [
-        {
-          "$ref": "#/definitions/Device"
+      "type": "object",
+      "properties": {
+        "device": {
+          "$ref": "https://raw.githubusercontent.com/ondrejtomcik/coreocf/Bugzilla-2709/swagger2.0/oic.wk.d.swagger.json#/definitions/Device"
         },
-        {
+        "devicestatus": {
```

Comment:
This information comes from the cloud. IMHO there might be more data added in near future, or extended by the company, therefore the root name of the property `devicestatus` does not fit. Also accessing field like `devicestatus.status` is pretty ugly. I would go for a different name - `properties`. `properties.status` to access `online`/`offline` information.

Reply:
I simplified it down to the single Property.  If we need to add more meta-information in the future then that's new Properties defined as appropriate.